src/status_im/subs.cljs

```diff
@@ -904,6 +904,13 @@
  (fn [[chat-id cursor]]
    (get cursor chat-id)))
 
+(re-frame/reg-sub
+ :chat/input-with-mentions
```

Comment:
what the difference between :chat/input-with-mentions and :chat**S**/input-with-mentions?

Reply:
`chats/input-with-mentions` is just a db map of `{chat-id data}` format, when `chat/input-with-mentions` is a chat specific `data`. I can rename it if that's confusing, just let me know what would look better here